packages/dids/src/did.methods/did.key.manager.ts

```diff
@@ -0,0 +1,48 @@
+import { driver } from 'did-method-key';
+import { IDidManager } from '../interface/i.did.manager';
+import {Wallet} from '@docknetwork/wallet-sdk-core/lib/modules/wallet';
+
+export class DidKeyManager  implements IDidManager{
+  private static instance: IDidManager;
+
+
+  private wallet;
+
+  private constructor() {
+    this.wallet = Wallet.getInstance();
+  }
+  async createDID(options = {}): Promise<string> {
+    const didDocument = await driver().generate();
+    return didDocument.id.toString()
+  }
+
+  async resolveDID(did: string): Promise<any> {
```

Comment:
this method should first check if the document exists in the wallet already or not as it can be used as a cache